index.html

```diff
@@ -11,151 +11,236 @@
-        </p>
-        <div class="detail three">
-          <ul class="nest">
-            <li><a href="#"><img src="image/Vector-(4).png" alt="Email icon" class="space"></a></li>
-            <li><a href="#"><img src="image/Linkedin-icon.png" alt="Social media" class="space"></a></li>
-            <li><a href="#"><img src="image/Vector-(1).png" alt="Email icon" class="space"></a></li>
-            <li><a href="#"><img src="image/Vector-(2).png" alt="Social media" class="space"></a></li>
+        <nav>
+          <ul class="menu-list">
+          <li><a href="#portfolio" class="web-header-link">porfolio</a></li>
+          <li><a href="#about" class="web-header-link">about</a></li>
+          <li><a href="#contact" class="web-header-link">contact</a></li>
           </ul>
-        </div>
-      </section>
-      <section class="cards">
-        <div class="card">
-          <img src="image/Snapshoot-Portfolio.png" alt="A photo of a website page">
-          <div class="card-content">
-            <h2 class="card-heading">Tonic</h2>
-            <ul class="tool1">
-              <li>CANOPY</li>
-              <li><img src="image/Counter.png" alt="dot"></li>
-              <li class="fint">Back End Dev</li>
-              <li><img src="image/Counter.png" alt="dot"></li>
-              <li class="fint">2015</li>
-            </ul>        
-            <p class="detail-style1">A daily selection of privately<br>reads:no account or<br>sign-up required</p>
-            <ul class="tool2">
-              <li><a href="#">html</a></li>
-              <li><a href="#">css</a></li>
-              <li><a href="#">javascript</a></li>
+          <img src="image/Union.png" alt="Menu icon" class="menu-logo">
+        </nav>
+      </header>
+      <section class="container-section">
+        <div class="container">
+          <div class="container-items">
+            <h1 class="detail one">
+              I'm Mariane <br> Glad to see you
+            </h1>
+            <p class="detail two">
+              I am a software developer! I can help you build a product, feature or website. Look through some of my work and experience. If you like what you see and have a project you need code, don't hesitate to contact me
+            </p>
+            <p class="two">
+              <a href="#">LET'S CONNECT</a>
+            </p>
+          </div>
+          <div class="detail three">
+            <ul class="nest">
+              <li><a href="#"><img src="image/Vector-(4).png" alt="Email icon" class="space"></a></li>
+              <li><a href="#"><img src="image/Linkedin-icon.png" alt="Social media" class="space"></a></li>
+              <li><a href="#"><img src="image/Vector-(1).png" alt="Email icon" class="space"></a></li>
+              <li><a href="#"><img src="image/Vector-(3).png" alt="Social media"></a></li>
+              <li><a href="#"><img src="image/Vector-(2).png" alt="Social media" class="space"></a></li>
             </ul>
```

Comment:
- Kindly add a `background-image` for the `hero` section in the desktop version to match the Figma design.

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/155534493-521ad73c-931f-45f7-8c3d-9ac244523f05.png) | ![Template 2-Main (1) - Copy](https://user-images.githubusercontent.com/24703171/155534555-d84801cb-7016-4048-b72f-f649a9ffcda3.png) |